build-tools/scripts/PrepareWindows.targets

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Prepare" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <_TopDir>$(MSBuildThisFileDirectory)\..\..</_TopDir>
+    <_NuGet>.nuget\NuGet.exe</_NuGet>
```

Comment:
Should this specify a directory?

Reply:
I only use `_NuGet` with `$(_TopDir)` as the `WorkingDirectory`, then I don't have to use relative paths to the sln files either.

I could do it a different way, though.